addons/website/static/src/js/set_view_track.js

```diff
@@ -0,0 +1,74 @@
+odoo.define('website.set_view_track', function (require) {
+"use strict";
+
+var CustomizeMenu = require('website.customizeMenu');
+var rpc = require('web.rpc');
+var Widget = require('web.Widget');
+
+var TrackPage = Widget.extend({
+    template: 'website.track_page',
+    xmlDependencies: ['/website/static/src/xml/track_page.xml'],
+    events: {
+        'change #switch-track-page': '_onTrackChange',
+    },
+    track: null,
+    start: function () {
+        var self = this;
+        this.$input = this.$('#switch-track-page');
+        this._is_tracked().then(function (data) {
```

Comment:
I'd say we can use arrowed functions from ES6 to be consistent with the decisions that were made.
So: `data => {` and then you don't need the `self` variable.